test/core/configuration/test_client_config.py

```diff
@@ -22,12 +22,16 @@
 
 class ClientConfigTest(unittest.TestCase):
     def test_client_configuration(self):
+        request_timeout_millis = 10_000
+        request_timeout_seconds = 10
```

Comment:
Let's calculate this from `request_timeout_millis` above.

Reply:
Done